recipes/bertrand/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+import os
+
+
+class SiConan(ConanFile):
+    name = "bertrand"
+    license = "LGPLv3"
```

Comment:
```suggestion
    license = "LGPL-3.0-or-later"
```

Follow short-term name https://spdx.org/licenses/LGPL-3.0-or-later.html

Reply:
Hmmm https://opensource.org/licenses/LGPL-3.0 uses a different one

Reply:
https://github.com/conan-io/conan-center-index/search?p=2&q=%22LGPL%22 Theres a bit too much variety

Reply:
LGPL-3.0 is deprecated, the new forms are LGPL-3.0-or-later and LGPL-3.0-only. There is a small section in the license which differs both. SDPX (short-term) is the correct pattern.